simpeg/fields.py

```diff
@@ -273,21 +348,73 @@ def __contains__(self, other):
 
 
 class TimeFields(Fields):
-    """Fancy Field Storage for time domain problems
-    .. code:: python
+    r"""Base class for storing TDEM fields.
+
+    ``TimeFields`` is a base class for storing discrete field solutions for simulations
+    that use discrete time-stepping; see :py:class:`SimPEG.simulation.BaseTimeSimulation`.
+    Generally only one field solution (e.g. 'eSolution', 'phiSolution', 'bSolution') is stored.
+    However, it may be possible to extract multiple field types (e.g. 'e', 'b', 'j', 'h')
```

Comment:
```suggestion
    Generally only one field solution (e.g. ``'eSolution'``, ``'phiSolution'``, ``'bSolution'``) is stored.
    However, it may be possible to extract multiple field types (e.g. ``'e'``, ``'b'``, ``'j'``, ``'h'``)
```